nav2_behavior_tree/include/nav2_behavior_tree/json_utils.hpp

```diff
@@ -0,0 +1,125 @@
+// Copyright (c) 2018 Intel Corporation
```

Comment:
I think this is your/company copyright :-)

Reply:
hahaha, this happened because I used the ```bt_utils.hpp``` as a template and just forgot to update the license :sweat_smile: